scripts/regenerate-playlists.ts

```diff
@@ -0,0 +1,8 @@
+import { Lobby } from '../lib/lobby';
+
+async function main() {
+  const lobbies = await Lobby.all();
+  lobbies.forEach(lobby => void lobby.synthesizePlaylist());
+}
+
+void main();
```

Comment:
is the void main so it runs every time the server restarts but only runs once per restart?

Reply:
The void is because main is async and returns a promise. Also this only runs whenever we run it!

Reply:
Oh yea but isnt the call to main() already executing the function?

Reply:
also wait can u add `this.managerName = props.managerName;` to the lobby constructor whoops
```suggestion
void main();
```

Reply:
Let’s do that in another PR